components/NotAuthenticatedMessage.tsx

```diff
@@ -17,7 +17,7 @@ const NotAuthenticatedMessage = (): JSX.Element => {
 
   return (
     <main className={styles.main}>
-      <Typography variant="h1">You must be logged in to see that...</Typography>
```

Comment:
Note that the header-level should not be chosen by how big it appears, but according to the structure of the document.

Example proper structure: https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#active_learning_exploring_the_code_for_our_example

Reply:
Good point.

I don't think I was looking at absolute size of the text though, but more accross different components that to me should have the same hierarchy. The message "You must be logged in to see that" was h1 whereas "You are logged in" in pages/index.tsx is h2.